src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -1,24 +1,56 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.*;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ALIAS;
 import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
 
 import seedu.address.model.Model;
+import seedu.address.model.person.*;
+import seedu.address.model.tag.Tag;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
 
 /**
  * Lists all persons in the address book to the user.
  */
 public class ListCommand extends Command {
 
-    public static final String COMMAND_WORD = "list";
+    public static final String COMMAND_WORD = "querystudents";
 
     public static final String MESSAGE_SUCCESS = "Listed all persons";
 
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists all students that meets the condition "
+            + "Optional Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_PHONE + "PHONE "
+            + PREFIX_EMAIL + "EMAIL "
+            + "[" + PREFIX_ALIAS + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "John Doe "
+            + PREFIX_PHONE + "98765432 "
+            + PREFIX_EMAIL + "johnd@example.com "
+            + PREFIX_ALIAS + "friends "
+            + PREFIX_ALIAS + "owesMoney";
```

Comment:
```suggestion
```

I'm assuming that you don't intend to allow the user to query by multiple aliases since patients can only have one alias, so just remove this line?

Reply:
Actually I was thinking it is fine cos 0 ... * alias gives us more flexibility next time if required.

Reply:
Hmm, a student can have 0...* aliases? Or do you mean that we can query by multiple aliases?

Reply:
A student can have zero or more alias. At this juncture we cannot query by multiple alias.